apps/ftu/js/ui.js

```diff
@@ -431,6 +431,9 @@ var UIManager = {
       case 'metrics-basic':
       case 'metrics-enhanced':
       case 'metrics-none':
+        this.updateSetting('hud.hide', (event.target.value === 'Enhanced'));
+        this.updateSetting('devtools.overlay',
```

Comment:
This is a nit, I think. I'm wondering if we should only update these settings when the level is enhanced. If ever devtools was enabled by default, we would disable it here if the level != enhanced. Now that I'm thinking about it, how about moving the 'enhanced' case to be first in the list and within the 'enhanced' block set 'hud.hide' and 'devtools.overlay' and then fall-through to the other cases where the metrics level is set.